test/test_THDMIIMSSMBCFull_approximation.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
@uukhas, shouldn't this be `/bin/sh`?

Reply:
For this file it does not matter. Kind of funny, that I made a decision between `sh` and `bash` based on the file `test/test_HSSUSY_SUSYHD.sh` - the **only** test file with `bash`.

Reply:
Ok. I'll change it to sh to make it more uniform.